metricbeat/module/docker/diskio/helper.go

```diff
@@ -45,56 +48,62 @@ func (io *BLkioService) getBlkioStatsList(rawStats []docker.Stat, dedot bool) []
 	return formattedStats
 }
 
-func (io *BLkioService) getBlkioStats(myRawStat *docker.Stat, dedot bool) BlkioStats {
+// TODO: User StorageStats instead of BlkioStats on Windows
```

Comment:
According to docker stats API documentation, `BlkioStats` is only populated in Linux, for Windows there is another structure under `StorageStats`.

Reply:
The biggest issue about Windows at the moment is testing it, maybe better to leave the TODO here and open a follow-up issue, yes.